app/Http/Controllers/Api/InventoryController2.php

```diff
@@ -0,0 +1,71 @@
+<?php namespace App\Http\Controllers\Api;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Item;
+use Illuminate\Support\Facades\Response;
+use Illuminate\Support\Facades\Input;
+
+class InventoryController2 extends Controller {
+
+	/**
+	 * Display the specified resource.
+	 *
+	 * @param  int  $tag
+	 * @return Json found $item
+	 */
+	public function show($tag)
+	{
+
+        $item=Item::where('asset_tag',$tag)->first();
+        return $item;
+
+	}
+
+
+	/**
+	 * Update the specified resource in storage.
+	 *
+	 * @param  int  $tag
+	 * @return Json updated $item
+	 */
+	public function update($tag)
+	{
+        $item=Item::where('asset_tag',$tag)->first();
```

Comment:
The indentation looks good in my local, but it's wried on git. I'm working on the display issue.